src/serializer.rs

```diff
@@ -1085,6 +1119,18 @@ mod tests {
         assert_eq!(got, "true,5,hi\n");
     }
 
+    #[test]
+    fn struct_no_headers_128() {
+        #[derive(Serialize)]
+        struct Foo {
+            x: i128,
+            y: u128,
+        }
+
+        let got = serialize(Foo { x: i128::max_value(), y: u128::max_value() });
+        assert_eq!(got, "170141183460469231731687303715884105727,340282366920938463463374607431768211455\n");
```

Comment:
Could you try to make sure code is wrapped to 79 columns inclusive? Thanks!

Reply:
I've wrapped these lines.